### PR TITLE
add name to images

### PR DESCRIPTION
**Pull request by rrhyne**
Hi, first... great module. Thanks!

I have cause to upload a specific set of images to a model. For instance, a 'front', 'side', 'top' and 'back' image. I needed to be able to recall those images by name to occupy certain locations in my views and to update them. 

This pull request allows me to do so, I hope you'll consider incorporating into the master!

Thanks again,

--rob

### CODE REVIEW

**Comment by CostaRico** [comment]
Thx a lot! I'll check your request out in several days and approve it!

File: models/Image.php [hunk 1]
```diff
@@ -280,6 +280,7 @@ public function rules()
         return [
             [['filePath', 'itemId', 'modelName', 'urlAlias'], 'required'],
             [['itemId', 'isMain'], 'integer'],
+            [['name'], 'string', 'max' => 80],
```
Comment (kotchuprik) [inline models/Image.php]: but in a database you want to store up to 255 symbols.

**Comment by rrhyne** [comment]
I'll get to that this weekend. Also, I think the dynamic where clause in the image finder should be parameratized for security.

**Comment by rrhyne** [comment]
Fixed the 80 character issue. Apologies for the delay.

**Comment by rrhyne** [comment]
Could we get this merged in?

**Comment by CostaRico** [comment]
Hi, guys, tell me pls, could anybody make some manually tests to check if everything works?

**Pull request merged**